grakn-test/src/test/java/ai/grakn/test/engine/controller/ImportControllerTest.java

```diff
@@ -137,9 +135,7 @@ private void runAndAssertCorrect(File file, List<String> findByResource, String
     }
 
     private void waitToFinish() {
-        final long initial = new Date().getTime();
-
-        while ((new Date().getTime())-initial < 2*60*60000) {
+        while (true) {
```

Comment:
SO correct me if I am wrong but if the end point is never reached (for some random failure) the tests will just timeout on travis and jenkins and we will be left confused. Is there no better way to handle this?